python/cuml/manifold/umap.pyx

```diff
@@ -357,8 +365,8 @@ class UMAP(Base):
 
         del state['handle']
 
-        cdef size_t params_t = <size_t>self.umap_params
-        cdef UMAPParams* umap_params = <UMAPParams*>params_t
+        cdef size_t params_t = <size_t > self.umap_params
```

Comment:
These random spaces in the umap cython seem unecessary. Can we revert all of these?

Reply:
I used autopep8 to auto-format Python code. Ok I'll remove these.